net_test.go

```diff
@@ -162,7 +162,7 @@ func TestListener(t *testing.T) {
 }
 
 func TestDial(t *testing.T) {
-	ln, err := Listen("tcp4", "127.0.0.1:0")
+	ln, err := Listen("tcp", ":0")
```

Comment:
That was like this some time before, until I started get errors on Travis CI.

Reply:
I noticed that travis is complaining. I wonder if it is a problem with travis then. Perhaps specifying a port can help. Testing.

Reply:
confident this was a bug in go interpreting/parsing IPv6 addresses that got fixed before 1.9. The issue probably started being noticeable when they defaulted travis to IPv6. But is just a guess, suggestion works now and is good enough for me too.